README.md

```diff
@@ -155,15 +155,26 @@ Container can be built via running `make docker-build`.
 
 ## Commitment Schemas
 
-Commitments returned from the EigenDA Proxy adhere to the following byte encoding:
+### Optimism Commitment Mode
+For `alt-da` clients running on Optimism, the following commitment schema is supported:
 
 ```
- 0        1        2         3               N
- |--------|--------|---------|---------------|
-  commit   da layer  version  raw commitment
+ 0        1        2         3                N
+ |--------|--------|---------|----------------|
+  commit   da layer  version   raw commitment
   type       type     byte
 ```
 
+### Generic Commitment Mode
```

Comment:
What does generic mean here? Is this just nitro? anything else uses this?

Reply:
anything that's not OP